source/docs/software/roborio-info/roborio-introduction.rst

```diff
@@ -0,0 +1,10 @@
+roboRIO Introduction
+====================
+
+The roboRIO is designed specifically with FIRST in mind. The roboRIO uses the same basic architecture of a Real-Time processors + FPGA as the previous cRIO but is more powerful, lighter, and smaller.
```

Comment:
"Previous cRIO" seems not helpful to the folks this article is aimed at. They don't really care what came before it I don't think.

Reply:
So this is the roboRIO page that is more in depth, so I don't mind it going to a little more detail but I agree that the past  probably doesn't matter to them and is a maintenance burden for us because it is constantly changing.